tests/flytekit/unit/core/test_node_creation.py

```diff
@@ -134,3 +134,30 @@ def t1(a: int) -> nt:
         def my_wf(a: int) -> str:
             t1_node = create_node(t1, a=a)
             return t1_node.outputs
+
+
+def test_runs_before():
+    @task
+    def t2(a: str, b: str) -> str:
+        return b + a
+
+    @task()
+    def sleep_task(a: int) -> str:
+        a = a + 2
+        return "world-" + str(a)
+
+    @dynamic
+    def my_subwf(a: int) -> (typing.List[str], int):
+        s = []
+        for i in range(a):
+            s.append(sleep_task(a=i))
+        return s, 5
+
+    @workflow
+    def my_wf(a: int, b: str) -> (str, typing.List[str], int):
+        subwf_node = create_node(my_subwf, a=a)
+        t2_node = create_node(t2, a=b, b=b)
+        subwf_node.runs_before(t2_node)
```

Comment:
This test will fail if we change this line to  `subwf_node >> t2_node`

Reply:
yeah, good point.  i'll change this in a bit.